clowdapp.yaml

```diff
@@ -142,11 +142,11 @@ objects:
         image: ${IMAGE}:${IMAGE_TAG}
         resources:
           limits:
-            cpu: ${CPU_LIMIT}
-            memory: ${MEMORY_LIMIT}
+            cpu: 250m
```

Comment:
Should we set these values in the parameters section (lines 309-316)?  If not, then we should remove the parameters if they're not being used.

Reply:
Removed parameters! Thanks